data-reconciliation-app/src/utils/dependencies.ts

```diff
@@ -0,0 +1,30 @@
+import {
```

Comment:
I'm not sure about pulling all dependencies in single file. Every module should decide their dependencies. Otherwise, this file will be a burden to maintain.

Reply:
This file act as Startup.cs , where all dependencies are injected

Reply:
I think there is no `Startup.cs` mentality in NodeJS. The module system and correspondingly the tree-shaking for bundling, reducing size etc are all built on simple conventions. The NodeJS module system doesn't require a Dependency Injection framework at all 😃

Reply:
Yeah, but more rich typescript frameworks like angular using Dependency Injection.

I will reomve it as you mention, and later we can have a deeper discussion.